File: Backend/WebApp.Service/Models/Configurations/JobBaseStatusModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using WebApp.Service.Models.Enrols;

namespace WebApp.Service.Models.Configurations
{
    public class JobBaseStatusModel : MasterModel
    {
        public long? CompanyId { get; set; }
        public long? BranchId { get; set; }
        public long? ProjectId { get; set; }
        public string JobBaseStatusCode { get; set; }
        public string JobBaseStatusTitle { get; set; }
        public CompanyInfoModel CompanyInfo { get; set; }
        public BranchInfoModel BranchInfo { get; set; }
        public ProjectModel Project { get; set; }
 
    }
}

File: src/app/feature/model/configurations/jobbasestatus.model.ts
import { CompanyInfo } from "../companyinfo.model";
import { Project } from './../project.model';

export interface JobBaseStatus{
    id:number,
    companyInfo:CompanyInfo,
    branchId:number,
    project:Project,
    jobBaseStatusCode:string,
    jobBaseStatusTitle:string
}